pkgs/applications/misc/st/default.nix

```diff
@@ -4,22 +4,20 @@
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
-  name = "st-0.6";
+  name = "st";
+  version = "0.7";
   
   src = fetchurl {
-    url = "http://dl.suckless.org/st/${name}.tar.gz";
-    sha256 = "0avsfc1qp8zvshsfjwwrkvk411jlqy58z225bsdhjkl1qc40qcc5";
+    url = "http://dl.suckless.org/st/${name}-${version}.tar.gz";
+    sha1 = "358bca5bfc6ee85352dcd4f45f13f78474e5de42";
   };
-
-  inherit patches;
+  patches = [ ./st.patch ];
```

Comment:
It makes the delete key work. At least on my computers it didn't work out of the box. The patch changes the ordering of + and - operators, I think this is a bug, somewhat related to http://lists.suckless.org/dev/1212/13800.html

Should I document this in the code somehow?